### DISCUSSION TITLE
Real World Data Science - Editor’s note: Not saying goodbye, just saying…

### DISCUSSION THREAD

**Discussion by giscus** [realworlddatascience/realworlddatascience.github.io#163] [General]
# Real World Data Science - Editor’s note: Not saying goodbye, just saying…

I’ll be leaving Real World Data Science and the Royal Statistical Society in a month’s time. Before I go, I want to say a big thank you to all the contributors, readers, staff and members that I’ve been fortunate to work with over the years.

https://realworlddatascience.net/viewpoints/editors-blog/posts/2024/03/editors-note.html

**Comment by Lextuga007** [comment]
Sorry to see you are leaving but excited for your next move! I've been very grateful for your input beyond RSS to help us at NHS-R Community with both the conference last year and for sharing all the code that went behind building this fabulous website. I truly hope our paths will cross again sometime! All the very best (and compliments to this comments field too - I really like it!). Zoë Turner

**Reply by brtarran** [reply]
Thanks, Zoë. Being part of the conference last year was great -- so much brilliant data science work going on across the NHS! Best of luck with your work on transferring the NHS-R Community site over to Quarto. I'm by no means an expert, but if I can help out at all, let me know.
